packages/atomic-hosted-page/src/pages/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+    <title>Coveo Atomic Hosted Page</title>
+
+    <script type="module" src="/build/atomic-hosted-page.esm.js"></script>
+    <script>
+      (async () => {
+        await customElements.whenDefined('atomic-hosted-page');
+
+        const page = document.querySelector('atomic-hosted-page');
+        await page.initialize({
```

Comment:
Eventually the feature will reach prod, I'll enable the feature flag for searchuisamples and add a search page there.
For now using dev